lib/config/config-validator.js

```diff
@@ -181,10 +186,10 @@ function formatErrors(errors) {
  * @returns {void}
  */
 function validateConfigSchema(config, source) {
-    const validator = schemaValidator(configSchema, { verbose: true });
+    validateSchema = validateSchema || ajv.compile(configSchema);
```

Comment:
@zxqfox is it better to make a commit now or make a separate PR after this one merged?